eGovPDX/portlandor
# POWR-2352 & POWR-2317: Moving social media links to contact content type and other contact related enhancements

This PR contains all of the development work and configuration for deprecating old social media fields, moving/adding social media fields to the contact content type, and ensuring the relationships created were able to render the results correctly.

**As an editor**
1. **When editing a contact item —>** the user should now see a "Social Media" group that contains the three existing social media fields (Facebook, Twitter, and Instagram) as well as three new ones (YouTube, LinkedIn, and NextDoor).
2. **When editing a either a Bureau/office, Elected official, or Program group —>** the user should see the old social media fields deprecated with a title and description
3. If a contact entity relationship is added, the render of that entity will supersede the deprecated fields output

**To Test:**
1. Log into https://powr-2352-portlandor.pantheonsite.io/
2. Add a new contact item —> upon save you will be able to see the new default fields templates
3. Edit an existing Bureau/office, Elected official, or Program group —> then add the new contact item created in step 1 as the relation
4. View the group page and see the new fields output
